bigquery/tests/unit/test_client.py

```diff
@@ -383,6 +383,35 @@ def test_create_dataset_w_attrs(self):
         self.assertEqual(ds.description, DESCRIPTION)
         self.assertEqual(ds.friendly_name, FRIENDLY_NAME)
 
+    def test_get_table(self):
+        project = 'PROJECT'
+        dataset_id = 'dataset_id'
+        table_id = 'table-id'
+        path = 'projects/%s/datasets/%s/tables/%s' % (
+            project, dataset_id, table_id)
+        creds = _make_credentials()
+        http = object()
+        client = self._make_one(project=project, credentials=creds, _http=http)
+        resource = {
+            'id': '%s:%s:%s' % (project, dataset_id, table_id),
+            'tableReference': {
+                'projectId': project,
+                'datasetId': dataset_id,
+                'tableId': table_id
+            },
+        }
+        conn = client._connection = _Connection(resource)
+        dataset_ref = client.dataset(dataset_id)
```

Comment:
dataset_ref = client.dataset(...).table(...)
?

Reply:
I think you mean `table_ref = client.dataset(...).table(...)`, but yeah +1 to that since it looks like dataset_ref is not used for anything else.

Reply:
I removed the unnecessary variable